### ISSUE TITLE
We would like to contribute to save money!

### ISSUE THREAD

**Issue by adir1** [exercism/exercism.io#6740]
As mentioned by few people on Hacker News - we are surprised the site is costing this much to host in this day and age.
I am currently in between jobs, but have over 30 years experience with software in large companies. I would recommend putting together a small focus group of experts to review larges areas of monthly costs and brainstorm design/vendor/code changes to reduce the costs.
Normally these days self-hosted APIs and DBs provide exceptional performance-to-price ratio, and most web assets should be on extremely-low-cost CDN hosting.
So to recap recommendation:
1. Setup task force to focus on cost cutting
2. Their output should be clearly documented roadmap/plan that can be posted/communicated to contributors
3. Divvy up tasks and issues to contributors to implement this critical transformation
Thank You!

**Comment by iHiD** [comment]
Hello. If you'd like to look at [our terraform](https://github.com/exercism/terraform/) and see where you think savings can be made, that'd be great.

Two points:
- We run a couple-of-hundred docker containers that have to be created/deleted whenever someone runs tests or we analyze their code, and that we rely heavily on EFS to distribute files across those machines.
- We have zero day-to-day ops support or "on call" stuff. So a server breaking and things going down as a result is unacceptable to us. We've had zero downtime on AWS for the last few years at that basis, so anything we changed to would need that same constraint.
